nav2_tasks/include/nav2_tasks/is_stuck_condition.hpp

```diff
@@ -91,14 +61,33 @@ class IsStuckCondition : public BT::ConditionNode, public rclcpp::Node
 
   ~IsStuckCondition()
   {
+    RCLCPP_DEBUG(this->get_logger(), "PlannerTester::~PlannerTester");
```

Comment:
The debug message here says PlannerTester. I think messages like "Shutting down IsStuckCondition" are more useful too, IMO

Reply:
Fixed